pkg/scaling/scale_handler.go

```diff
@@ -348,7 +354,8 @@ func (h *scaleHandler) getPods(scalableObject interface{}) (*corev1.PodTemplateS
 
 		if withPods.Spec.Template.Spec.Containers == nil {
 			h.logger.Info("There aren't any containers in the ScaleTarget", "resource", obj.Status.ScaleTargetGVKR.GVKString(), "name", obj.Spec.ScaleTargetRef.Name)
-			return nil, "", fmt.Errorf("no containers found")
+			h.logger.Info("You may not be able to use all scaler types")
```

Comment:
No need to add a second log message, we should update the one above to message like this:

`There aren't any containers found in the ScaleTarget, therefore it is no possible to inject environment properties`

Reply:
And this log message should be probably debug one: `h.logger.V(1).Info(...)`

Reply:
Yeah that's better fixed.